<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergyMonitor</class>
 <widget class="QWidget" name="EnergyMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layout_main">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Power Factor:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Voltage:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="FixedSizeLabel" name="label_current">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="FixedSizeLabel" name="label_voltage">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="FixedSizeLabel" name="label_power_factor">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10" colspan="2">
        <widget class="QPushButton" name="button_energy">
         <property name="text">
          <string>Reset Energy</string>
         </property>
        </widget>
       </item>
       <item row="2" column="10">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Transformer Current Ratio:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="11">
        <widget class="QDoubleSpinBox" name="spinbox_current_ratio"/>
       </item>
       <item row="3" column="10" colspan="2">
        <widget class="QPushButton" name="button_transformer_settings">
         <property name="text">
          <string>Save Transformer Settings</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="FixedSizeLabel" name="label_frequency">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Transformer Voltage Ratio:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Frequency:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Wh</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="FixedSizeLabel" name="label_real_power">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QDoubleSpinBox" name="spinbox_voltage_ratio"/>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Real Power:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="FixedSizeLabel" name="label_energy">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="12">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Energy:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9" rowspan="4">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Apparent Power:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="FixedSizeLabel" name="label_reactive_power">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="FixedSizeLabel" name="label_apparent_power">
         <property name="text">
          <string>TBD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>VA</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>var</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Reactive Power:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_graph">
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FixedSizeLabel</class>
   <extends>QLabel</extends>
   <header location="global">brickv.fixed_size_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
